### PR TITLE
[#7208] styling: userdashboard: fix project list tiles and tab location, clean comp li…

### PR DESCRIPTION
**Pull request by philli-m**
…b and rm uneeded styles

### CODE REVIEW

File: adhocracy-plus/assets/scss/components/_filter.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.filter--full {
+    margin-top: $spacer;
+
+    & button {
+        width: 100%;
+    }
+}
```
Comment (philli-m) [inline adhocracy-plus/assets/scss/components/_filter.scss:7]: I would like to move all filter styling to here eventually so starting now

File: adhocracy-plus/templates/a4dashboard/blueprint_list.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
-{% extends "a4dashboard/base_dashboard.html" %}
+<!-- {% extends "a4dashboard/base_dashboard.html" %}
```
Comment (khamui) [inline adhocracy-plus/templates/a4dashboard/blueprint_list.html:1]: is a closing comment missing here?

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx [hunk 3]
```diff
@@ -161,42 +161,52 @@ export default class ModerationNotificationList extends Component {
...
+        </div>
+
+        <div
+          className="row"
+          aria-label={django.pgettext('kosmo', 'Filter')}
+        >
+          <div className="col-md">
             <Filter
-              filterClass="mt-3 mt-lg-5 me-lg-3 dropdown dropdown-menu-end"
+              filterClass="filter--full dropdown dropdown-menu-end"
```
Comment (khamui) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:183]: out of curiosity: is there a general rule we follow when deciding, when to use bs and when to use our own classes?

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx [hunk 4]
```diff
@@ -255,8 +257,8 @@ export const ModerationNotification = (props) => {
   }
 
   return (
-    <>
-      <li className="list-item">
+    <li className="list-item border-0 p-0">
+      <div className="list-item">
```
Comment (khamui) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:261]: is this div still needed and should it use the same class `list-item` as its parent?

**Review by khamui — COMMENTED** [review]
the three columns spacing looks a little bit off (spacing to bottom is too less? and to the right edge is too big?)

And the image looks a little bit wider than in the design (nit-picky, but I thought I mention it)

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx (hunk 3 above)
Comment (philli-m) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx:183]: @khamui that's a good question! the answer is kind of but not really, in this case the BS classes `dropdown dropdown-menu-end` are needed for the functionality so unless it is general styling that should be applied to every dropdown we should not add anything to that class so having an additional class is preferable. Also as in this case we will want all children of filter--full to be full width, we add a class as this is simpler then assigning bs classes in multiple places. I don't think it's particularly standard to have such a mix as we do and finding the right medium is pretty hard but as bootstrap can be very long winded to get something precise it's kind of if it 'feels' right then we go with bs but if not with a component class.

File: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx (hunk 4 above)
Comment (philli-m) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:261]: the div is needed to ensure that the mod statement and comment are in the same border box but the comment box for the statement is external as in the designs but you are right, double use of list-item is not great, i'll go back and reconsider

**Comment by khamui** [comment]
lgtm!

**Pull request merged**